charts/microgateway/README.md.gotmpl

```diff
@@ -376,6 +377,21 @@ Finally, apply the Helm chart configuration file with `-f` parameter.
   helm upgrade -i microgateway airlock/microgateway -f custom-values.yaml -f env-variables.yaml
   ```
 
+## Extra Volumes
+The Helm Chart allows you to define extra volumes which can then be used in the Microgateway. 
+The configuration of such additional volumes could look like this, for example: 
+
+```
+extraVolumes:
+    - name: mapping
+      configMap:
+        name: mapping-configmap
+extraVolumeMounts:
+    - name: mapping
+      mountPath: /var/config/template/mapping.xml
```

Comment:
> mountPath: /var/config/template/mapping.xml
The default mapping.xml is expected under /config/mapping.xml.
I suggest to adapt the example slightly. Change the path to /config/templates/mapping.xml

Maybe it would be great to also show the necessary part of the DSL configuration. What do you think?